package.json

```diff
@@ -665,6 +666,7 @@
     "style-loader": "^0.21.0",
     "stylelint": "^13.6.1",
     "superstruct": "^1.0.3",
+    "tailwindcss": "^3.3.0",
```

Comment:
Adding tailwind as a dependency. v4 has been release but we need to check compatibility with the design system preset and other platforms that use tailwind e.g. portfolio